src/pages/bridge/ConfirmDialog/index.tsx

```diff
@@ -0,0 +1,92 @@
+import { useEffect, useMemo, useState } from "react"
+
+import Button from "@mui/material/Button"
+import Checkbox from "@mui/material/Checkbox"
+import Dialog from "@mui/material/Dialog"
+import DialogActions from "@mui/material/DialogActions"
+import DialogContent from "@mui/material/DialogContent"
+import DialogContentText from "@mui/material/DialogContentText"
+import DialogTitle from "@mui/material/DialogTitle"
+import FormControlLabel from "@mui/material/FormControlLabel"
+import { styled } from "@mui/system"
+
+import Link from "@/components/Link"
+
+const StyledDialog = styled(Dialog)(({ theme }) => ({
+  "& .MuiDialog-paper": {
+    width: "60rem",
+    padding: "1rem 2.4rem 2.4rem 1rem",
+  },
+}))
+
+const StyledLink = styled(Link)(({ theme }) => ({
+  color: "inherit",
+  textDecoration: "underline",
+  "&:hover": {
+    opacity: 0.85,
+  },
+}))
+
+const ConfirmDialog = ({ open, setOpen, send }) => {
+  const [checkbox1Selected, setCheckbox1Selected] = useState<boolean>(false)
+  const [checkbox2Selected, setCheckbox2Selected] = useState<boolean>(false)
+
+  const selectedAll = useMemo(() => {
+    return checkbox1Selected && checkbox2Selected
+  }, [checkbox1Selected, checkbox2Selected])
+
+  useEffect(() => {
+    if (!open) {
+      setCheckbox1Selected(false)
+      setCheckbox2Selected(false)
+    }
+  }, [open])
+
+  const handleClose = () => {
+    setOpen(false)
+  }
+
+  const handleSendTransaction = () => {
+    setOpen(false)
+    send()
+  }
+
+  return (
+    <StyledDialog maxWidth={false} open={open}>
+      <DialogTitle sx={{ fontSize: "2.4rem", mb: "2rem", fontWeight: 600 }}>Move funds to Goerli</DialogTitle>
+      <DialogContent>
+        <DialogContentText sx={{ mb: "2rem", fontWeight: 500 }}>
+          <StyledLink target="_blank" href="http://localhost:3000/bridge#end" rel="noopener noreferrer">
```

Comment:
relative path is better

Reply:
could replace `target="_blank"` with `external` , remove `rel="noopener noreferrer"`, and add `underline="always"` when using `@/components/Link`